lib/resolveStepDefinition-another.test.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable global-require */
+/* global jest */
+const fs = require("fs");
+const { Parser } = require("gherkin");
+const { createTestsFromFeature } = require("./createTestsFromFeature");
+const {
+  defineParameterType,
+  defineStep,
+  when,
+  then,
+  given,
+  and,
+  but
+} = require("./resolveStepDefinition");
+
+window.defineParameterType = defineParameterType;
+window.when = when;
+window.then = then;
+window.given = given;
+window.and = and;
+window.but = but;
+window.defineStep = defineStep;
+window.cy = {
+  log: jest.fn()
+};
+
+window.Cypress = {
+  env: jest.fn()
+};
+
```

Comment:
I think we could move all this setup to a separate file in this case, and require it in both test files, to remove the duplication. Also, that file could export readAndParseFeatureFile